### PR TITLE
added prompt template

### PR DESCRIPTION
**Pull request by ConanChou**
- added an option for user to tweak the prompt
- formatted the ChatGPTAPI output, so it match the style with GPT3 and Google

### CODE REVIEW

**Comment by yihong0618** [comment]
@ConanChou Thanks LGTM
do we need add one test in CI?

**Comment by ConanChou** [comment]
re: test - yeah, makes sense, didn't realize we have test in the workflow.
re: #144 - i can take a look tomorrow. it's getting late in new york 😴

File: book_maker/cli.py [hunk 1]
```diff
@@ -90,6 +107,13 @@ def main():
         default=False,
         help="allow NavigableStrings to be translated",
     )
+    parser.add_argument(
```
Comment (zstone12) [inline book_maker/cli.py:110]: maybe add some param check to cofirm --prompt contains  {text} & {language} could be better?

**Comment by zstone12** [comment]
LGTM.The conflict needs to be resolved:D

**Comment by yihong0618** [comment]
> re: test - yeah, makes sense, didn't realize we have test in the workflow. re: #144 - i can take a look tomorrow. it's getting late in new york 😴

good night~

**Comment by yihong0618** [comment]
merge first~
Thanks a lot ~

**Comment by ConanChou** [comment]
oh cool! thanks, I will continue to make finishing up the suggested changes

**Comment by yihong0618** [comment]
thanks a lot

**Comment by ConanChou** [comment]
actually I think you have done the above suggested changes. thanks!

I plan to unify the way we set the `system` and `user` prompt, making both configurable via using `--prompt` and system env. Now it's a bit weird.

**Pull request merged**